docker/stringsifter/pyproject.toml

```diff
@@ -0,0 +1,18 @@
+[tool.poetry]
+name = "dockerfiles"
+version = "0.1.0"
+description = ""
+authors = ["TalZich <tzichlinsky@paloaltonetworks.com>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "~3.10"
+stringsifter = "3.20230711"
+wheel = "*"
```

Comment:
why do we need wheel?

Reply:
when trying to `pip install stringsifter` on my local machine, a `ModuleNotFoundError` kept popping up saying `No module named wheel`. I figured it's a requirement for stringsifter. should it not be here?

Reply:
lets remove from the poetry and see if it comes with the lock